inductiva/coastal/README.md

```diff
@@ -8,10 +8,76 @@ is performed using the [SWASH](https://swash.sourceforge.io/) simulator.
 
 ### Example
 
-Start by creating or loading a bathymetry. This can be done e.g. by reading an
+Note: You can follow this example in a notebook or run the script
+`random_bathymetry_demo.py` available in this folder, that contains all the code
+snippets exhibited here. Don't forget to insert your API key for the script to work!
+
+The bathymetry that we use can be loaded from a real scenario or randomly generated
+by us. To avoid having to download any data for now, we will start by run a simulation
+on synthetic bathymetry , that is procedurally generated using the API.
+
+We start by generating a random bathymetry.
+```python
```

Comment:
Let's have a fully working python code snippet from end-to-end with bathymetry generation and simulation. Hence, people can copy paste it.

Explain the current sections between code in a comment above the line of code.